src/core/icons/fileIcons.ts

```diff
@@ -2467,5 +2467,9 @@ export const fileIcons: FileIcons = {
       name: 'grafana-alloy',
       fileExtensions: ['alloy'],
     },
+    {
+      name: 'markdownlint',
+      fileNames: ['.markdownlint.json'],
```

Comment:
We can also add the following fileNames:
`.markdownlint.jsonc`, `.markdownlint.yaml`, `.markdownlint.yml`, `.markdownlint-cli2.jsonc`, `.markdownlint-cli2.yaml`, `.markdownlint-cli2.cjs`, `.markdownlint-cli2.mjs`

Ref: https://github.com/igorshubovych/markdownlint-cli#configuration, https://github.com/DavidAnson/markdownlint-cli2#configuration

Reply:
Okay, thanks